### ISSUE TITLE
Android builds fails with clean flutter 2.2 project

### ISSUE THREAD

**Issue by Moncader** [plaidev/karte-flutter#5]
Simply including karte_core in to pubspec.yaml causes the build to fail.

### Steps to reproduce
1. flutter create myproject
2. Add karte_core: 0.1.0 to dependencies of pubspec.yaml
3. flutter build apk

### Build error
```
e: /xxxxxxx/source/flutter-2.2/.pub-cache/hosted/pub.dartlang.org/karte_core-0.1.0/android/src/main/kotlin/io/karte/flutter/core/KarteCorePlugin.kt: (135, 48): Unresolved reference: VERSION_NAME
                                                                        
FAILURE: Build failed with an exception.
```

The reason for this is Flutter has updated to gradle 4+, which does not expose the VERSION_NAME to plugins.

**Comment by wasnot** [comment]
@Moncader 

Thanks for your information.

As you pointed out, with Android Gradle Plugin 4.1.0 or later, library project will no longer generate VERSION_NAME etc., so it is not possible to build.
We will fix it, so please wait. 

If you are in a hurry, please change to Android Gradle Plugin 4.0.0 etc. and build it. 
We apologize for the inconvenience.

**Comment by wasnot** [comment]
Fix karte_core: 0.1.1 has been released.
